src/hooks/useFarmStats.js

```diff
@@ -51,16 +52,21 @@ export function useFarmStats(farmAddress) {
           totalUnlockedRewards, 
         }] = result.data.tokenGeysers
 
+        // Calculate unlocked rewards per month.
         const unlockRate = totalUnlockedRewards / 30;
         
+        // Get today's date
         const today = new Date();
+        
+        // Calculate farm end date based on the bonus period (at this time, all rewards are unlocked)
         const farmEndDate = new Date((createdTimestamp + bonusPeriodSec) * 1000)
         const farmTimeLeft = farmEndDate - today;
 
+        // Return days left, if any (otherwise, remains at 0)
         if (farmTimeLeft > 0) {
           farmDaysLeft = Math.ceil(farmTimeLeft / (1000 * 60 * 60 * 24)); 
```

Comment:
Hmmm, I'm not too too sure this will remain at 0... `ceil` will round up, so even if we have something like `0.13` it will still become 1. Could we test this out?

Thinking about this a bit more, it might be the case than when there's less than 1 day left, we might just wanna get hours. I don't have a good API for this hook in mind for now, but we might wanna handle this edge case better.